.github/workflows/constraints.txt.rej

```diff
@@ -0,0 +1,11 @@
+diff a/.github/workflows/constraints.txt b/.github/workflows/constraints.txt	(rejected hunks)
```

Comment:
No `.rej`-files should be commited to git. Remove all `.rej`-files from git with `git rm`.